libc/socket/unix-socket.c

```diff
@@ -475,9 +476,14 @@ void unix_msg_data_and_fd(int type)
 }
 
 
-TEST(test_unix_socket, unix_msg_data_and_fd)
+TEST(test_unix_socket, unix_stream_sock_send_and_fd)
```

Comment:
I think that we can remove this `unix` prefix as it's mentioned in the name of test group. Then we can name it `msg_data_and_fd_strem`. The current name is misleading - send and file descriptor, send what (?), also it's not only sending, but receiving too. IMHO it's better to keep the previous convention, where we have test cases like `data_only` and `data_and_fd`

Reply:
We can name the same way the rest of changed cases.